docs/cookbook/entities/configure-admin-menu.rst

```diff
@@ -0,0 +1,72 @@
+How to configure backend menu
+===============================
```

Comment:
It seems to have to much "=". It needs to be identical with the title length.

Reply:
Oh yes. I missed it.